libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -149,6 +165,7 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Description: Controls how often the GPS should provide a position update. Lowering below 5Hz is not allowed
     // @Units: milliseconds
     // @Values: 100:10Hz,125:8Hz,200:5Hz
+    // @Range: 100 200
```

Comment:
Are there actually GPSs that have a 20Hz update rate?  I don't think I've ever seen one.  In any case, I've now set the lower limit on this parameter description to 50.